syncto/views/collection.py

```diff
@@ -64,9 +64,7 @@ def collection_get(request):
     records = sync_client.get_records(collection_name, full=True,
                                       headers=headers, **params)
 
-    statsd = request.registry.statsd
-    if statsd:
-        statsd.count("syncclient.status_code.200")
+    status_code(request,str="syncclient.status_code.200")
```

Comment:
You need to import the command first.

The name ``status_code`` doesn't sounds like a good name for this, what about ``statsd_count(request, "syncclient.status_code.200")``

For the PEP8 you also need a space between the comma and the attribute.

str is not a right attribute name because it overtide the python str type.